src/components/CallToAction/FormSubmissionModal.js

```diff
@@ -62,8 +66,10 @@ const FormSubmissionModal = ({
     </span>
   ),
 }) => {
-  let followOnContent = <NewsletterAndTwitterInner siteMetadata={siteMetadata} />;
-  if (followOn === 'TWITTER') {
+  let followOnContent;
+  if (followOn === 'NEWSLETTER_AND_TWITTER') {
+    followOnContent = <NewsletterAndTwitterInner siteMetadata={siteMetadata} />;
+  } else if (followOn === 'TWITTER') {
     // Doesn't make sense to offer to let people sign up to the newsletter immediately after
```

Comment:
The changes in this file basically make it possible to not have any "follow on" text in the modal.